src/components/tile-select/tile-select.test.js

```diff
@@ -306,6 +306,31 @@ context("Tests for TileSelect component", () => {
       }
     );
 
+    it("p should be rendered when a description is passed to TileSelectGroup component", () => {
```

Comment:
Can we update these to be a little clearer:

`p element ...` or `group description container ...` etc